Phase04/SimpleCalculator.Test/SimpleCalculatorTest.cs

```diff
@@ -0,0 +1,24 @@
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.Enums;
+
+namespace SimpleCalculator.Test;
+
+public class SimpleCalculatorTest
+{
+    [Theory]
+    [InlineData(3, 4, OperatorEnum.sum, 7)]
+    [InlineData(3, 4, OperatorEnum.sub, -1)]
+    [InlineData(3, 4, OperatorEnum.division, 0)]
+    [InlineData(3, 4, OperatorEnum.multiply, 12)]
+    public void CalculateTest(int first, int second, OperatorEnum operatorEnum, int expected)
```

Comment:
- You're only testing Calculate function of Calculator object, test the program unit by unit
- This is not 'unit' testing since you're testing EVERYTHING in a single test
- Calculator should be the class's field and only be instantiated once